src/github.com/getlantern/bandwidth/bandwidth.go

```diff
@@ -0,0 +1,80 @@
+package bandwidth
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/getlantern/golog"
+)
+
+var (
+	log = golog.LoggerFor("bandwidth")
+
+	quota *Quota
+	mutex sync.RWMutex
```

Comment:
Seems like we should be able to skip both of the above to me. If we do really want the hello function, couldn't we just fetch the data from the channel there as well?